### DISCUSSION TITLE
Cutpoints not correct after export

### DISCUSSION THREAD

**Discussion by mendler2000** [mifi/lossless-cut#1094] [Support] [answered]
Hello, everyone,

great tool!

One problem: All catpoints fit in the preview. However, after exporting and merging, the segments start a few seconds earlier.

Is there a way to adjust this better?

**Comment by mifi** [comment, marked as answer]
Hi. Please see the main readme: https://github.com/mifi/lossless-cut#known-issues--limitations

**Reply by mendler2000** [reply]
Thank you!
